app/policies/comment_policy.rb

```diff
@@ -0,0 +1,25 @@
+class CommentPolicy
+  attr_reader :current_user, :comment
+
+  def initialize(current_user, comment)
+    @current_user = current_user
+    @comment = comment
+  end
+
+  def index?
+    @current_user.admin?
+  end
+
+  def create?
+    @current_user.admin?
+  end
+
+  def update?
+    @current_user.admin?
```

Comment:
I think admin can update only his personal comments, not comments created by other admins.

Reply:
Policy is not updated.